apps/data/priv/repo/seeds.exs

```diff
@@ -9,3 +9,50 @@
 #
 # We recommend using the bang functions (`insert!`, `update!`
 # and so on) as they will fail if something goes wrong.
+
+project_params_1 = %{
+  name: "Elixir School",
+  tags: ["elixir"],
+  url: "https://github.com/elixirschool/elixirschool",
+}
+
+project_params_2 = %{
+  name: "Extracurricular",
+  tags: ["elixir", "open source"],
+  url: "https://github.com/elixirschool/extracurricular",
+}
+
+project_params_3 = %{
+  name: "Appendix",
+  tags: ["elixir", "open source", "blog"],
+  url: "https://github.com/elixirschool/appendix",
+}
+
+{:ok, elixir_school}   = Data.Projects.insert(project_params_1)
+{:ok, extracirricular} = Data.Projects.insert(project_params_2)
+{:ok, appendix}        = Data.Projects.insert(project_params_3)
+
+opportunity_params_1 = %{
+  title: "Blog post on Git Bisect",
```

Comment:
How about we use something that's likely to be covered by an Elixir blog instead?

"Understanding GenServers and State"